src/routes/game/components/CannotCounterDialog.vue

```diff
@@ -46,9 +46,19 @@
     </template>
 
     <template #actions>
+      <v-btn
+        data-cy="cannot-counter-counter"
+        variant="flat"
+        disabled
```

Comment:
Note that because this is the `CannotCounterDialog`, this button is _permanently_ disabled